emmet/vasp/task_tagger.py

```diff
@@ -105,7 +118,7 @@ def task_type(inputs, include_calc_type=True):
     except Exception:
         functional = "PBE"
 
-    METAGGA_TYPES = {"TPSS", "RTPSS", "M06L", "MBJL", "SCAN", "MS0", "MS1", "MS2"}
+    METAGGA_TYPES = {"TPSS", "RTPSS", "M06L", "MBJL", "SCAN", "R2SCAN", "MS0", "MS1", "MS2"}
```

Comment:
The logic in `task_type` seems brittle to me in that it depends on us defining what functionals to expect. For example, as written it would return 'GGA+U' for a SCAN+U calc, and `GGA` for a calc using PBE potcars with an unknown functional like RSCAN. 

Can / should we re-do this in a way that's more resilient to future changes? For example, can we just call `Vasprun.run_type` from pymatgen?

See https://github.com/materialsproject/pymatgen/pull/1996